R/wrappers.R

```diff
@@ -8,7 +8,7 @@ setcoalesce = function(...) .Call(Ccoalesce, list(...), TRUE)
 fifelse = function(test, yes, no, na=NA) .Call(CfifelseR, test, yes, no, na)
 fcase   = function(..., default=NA) .Call(CfcaseR, default, parent.frame(), as.list(substitute(list(...)))[-1L])
 
-colnamesInt = function(x, cols, check_dups=FALSE) .Call(CcolnamesInt, x, cols, check_dups)
+colnamesInt = function(x, cols, check_dups=FALSE, source = "Unknown") .Call(CcolnamesInt, x, cols, check_dups, source)
```

Comment:
```suggestion
colnamesInt = function(x, cols, check_dups=FALSE, source = "") .Call(CcolnamesInt, x, cols, check_dups, source)
```

Unknown does not provide more information